src/trackedit/internal/trackedituiactions.cpp

```diff
@@ -115,7 +115,44 @@ const UiActionList TrackeditUiActions::m_actions = {
              TranslatableString("action", "Duplicate"),
              TranslatableString("action", "Duplicate")
              ),
-    // clip
+    // track actions
+    UiAction("track-rename",
```

Comment:
no longer needed

Reply:
@Eism UiAction is needed to populate the menu; please correct me if I'm wrong.

Reply:
in the case of track-rename, we have to create MenuItem manually and add it to the list

Reply:
@Eism @kryksyh to understand why this item is created manually and not from an action?